data/userRepository.js

```diff
@@ -0,0 +1,20 @@
+class UserRepository {
+  constructor(data){
+    this.usersData = data
+  }
+  retrieveUserInfo(id){
+    let userInfo = this.usersData.filter(user =>
+    user.id == id)
+    return userInfo
+  }
+  calculateAverageStepGoalAll(){
+    let totalStepGoals = this.usersData.reduce(((acc, user) =>
```

Comment:
I am a big advocate of naming the accumulator to something different than `acc`, I think this a refactor opportunity :)